modules/View.js

```diff
@@ -352,8 +389,337 @@
+          feedbackModelField = this.__substituteIndicesUsingMap(to, indexMap);
+        }
+        result = feedbackToInvoke.then.call(this, evt, indexMap);
+        this.__processFeedbackThenResult(result, feedbackModelField);
+      }
+    },
+
+    /************** Private methods **************/
+
+    /**
+     * Generates callbacks for changes in feedback model fields
+     * 'change fullName' -> invokes all the jQuery (or $) methods on the element as stored by the feedback model
+     * If feedbackModel.get('fullName') returns:
+     * { text: 'my text',
+     *   attr: {class: 'newClass'}
+     *   hide: [100, function() {...}]
+     * ...}
+     * Then it will invoke $element.text('my text'), $element.attr({class: 'newClass'}), etc.
+     * @private
+     * @method __generateFeedbackModelCallbacks
+     */
+    __generateFeedbackModelCallbacks: function() {
+      var self = this;
+      // Feedback one-way bindings
+      self.feedbackModel.off();
+      _.each(this.$('[data-feedback]'), function(element) {
+        var attr = $(element).data('feedback');
+        self.feedbackModel.on('change:' + attr, (function(field) {
+          return function() {
+            var $element,
+              state = self.feedbackModel.get(field);
+            if (!state) {
+              return;
+            }
+            $element = self.$el.find('[data-feedback="' + field + '"]');
+            _.each(state, function(value, key) {
+              var target;
+              if (_.first(key) === '_') {
+                target = self[key.slice(1)];
+              } else {
+                target = $element[key];
+              }
+              if (_.isArray(value)) {
+                target.apply($element, value);
+              } else if (value !== undefined) {
+                target.call($element, value);
+              }
+            });
+          };
+        })(attr));
+      });
+      _.each(self.feedbackModel.attributes, function(value, attr) {
+        self.feedbackModel.trigger('change:' + attr);
+      });
+    },
+
+    /**
+     * Processes the result of the then method. Adds to the feedback model.
+     * @param result {Object} the result of the then method
+     * @param feedbackModelField {Object} the name of the feedbackModelField, typically the "to" value.
+     * @private
+     * @method __processFeedbackThenResult
+     */
+    __processFeedbackThenResult: function(result, feedbackModelField) {
+      var newState = $.extend({}, result);
+      this.feedbackModel.set(feedbackModelField, newState, {silent: true});
+      this.feedbackModel.trigger('change:' + feedbackModelField);
+    },
+
+    /**
+     * Creates the "when" bindings, and collates and invokes the "then" methods for all feedbacks
+     * Finds all feedback zones that match the "to" field, and binds the "when" events to invoke the "then" method
+     * @private
+     * @method __generateFeedbackBindings
+     */
+    __generateFeedbackBindings: function() {
```

Comment:
This method is long.  I know that it's a port from FormView... but it's still too long.
Let's refactor this (not in this commit) to be a bit more readable.